values-tr/strings.xml

```diff
@@ -2493,106 +2494,106 @@ Press a key to assign it to this action.</string>
     <string name="ci_neighbor_ne">NE</string>
     <string name="ci_neighbor_nw">NW</string>
     <string name="ci_neighbor_sw">SW</string>
-    <string name="dialog_removewithprice_title">Removal costs</string>
+    <string name="dialog_removewithprice_title">Kaldırma bedeli</string>
     <string name="dialog_removewithprice_text">The price to remove your selection is %1$s. Do you want to proceed?</string>
 
 
     <string name="featureselector_doubledincometime">Double income for %1$s</string>
-    <string name="draft_marker_powermarker_text">Coverage with energy. Energy is transported using wires.</string>
-    <string name="draft_marker_waterpipemarker_title">Water</string>
-    <string name="draft_marker_waterpipemarker_text">The coverage with water. Water is transported to your residents using pipes.</string>
-    <string name="stage_files_title">File manager</string>
-    <string name="stage_files_importhere">Import here</string>
-    <string name="stage_files_currentdir">Current folder</string>
-    <string name="stage_files_actions">Actions</string>
+    <string name="draft_marker_powermarker_text">Coverage with energy. Enerji kablolar ile transfer edilir.</string>
+    <string name="draft_marker_waterpipemarker_title">Su</string>
+    <string name="draft_marker_waterpipemarker_text">The coverage with water. Su yerleşimcilerinize borular ile ulaştırılır.</string>
+    <string name="stage_files_title">Dosya yöneticisi</string>
+    <string name="stage_files_importhere">Buraya yükle</string>
+    <string name="stage_files_currentdir">Şu anki klasörr</string>
+    <string name="stage_files_actions">Eylemler</string>
     <string name="stage_files_actionexport">Export</string>
-    <string name="stage_files_actionrename">Rename</string>
-    <string name="stage_files_actionrename_text">Rename file %1$s:</string>
-    <string name="stage_files_renamecollision_title">Naming collision</string>
-    <string name="stage_files_renamecollision_text">The file %1$s does already exist!</string>
-    <string name="stage_files_actiondelete">Delete</string>
-    <string name="stage_files_actiondelete_title">Delete?</string>
+    <string name="stage_files_actionrename">Yeniden adlandır</string>
+    <string name="stage_files_actionrename_text">Dosyayı yeniden adlandır %1$s:</string>
+    <string name="stage_files_renamecollision_title">İsim çakışması</string>
+    <string name="stage_files_renamecollision_text"> %1$s isimli dosya zaten var!</string>
```

Comment:
I'm not sure you've intended this whitespace

Reply:
Sorry it's not my intention to leave whitespace

Reply:
I just want to clarify something. You are talking about whitespace at the beginning of sentences right? from 2513 and 2532

Reply:
Correct.